project/Common.scala

```diff
@@ -47,7 +47,7 @@ object Common extends AutoPlugin {
     projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
     crossVersion := CrossVersion.binary,
     crossScalaVersions := Dependencies.ScalaVersions,
-    scalaVersion := Dependencies.Scala213,
+    scalaVersion := Dependencies.Scala212,
```

Comment:
I would prefer to use Scala213 as the default. All I'm asking for is that we github workflow uses `target/scala-2.13`

Reply:
main pekko repos default to scala 2.13 - it's only a few like pekko-management that default to 2.12.

Reply:
Okay sure, ill revert this and update pekko management

Reply:
Done and pushed.